src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/Switch.cs

```diff
@@ -178,6 +185,17 @@ protected string Value
             }
         }
 
+        /// <summary>
+        ///  Occurs when a <see cref="Switch"/> needs to be initialized.
+        /// </summary>
+        public static event EventHandler<InitializingSwitchEventArgs>? Initializing;
+
+        internal void OnInitializing()
+        {
+            Initializing?.Invoke(null, new InitializingSwitchEventArgs(this));
+            TraceUtils.VerifyAttributes(Attributes, GetSupportedAttributes(), this);
```

Comment:
This call causes Winforms test to fail - https://github.com/dotnet/winforms/pull/7578